pcm_util.spec

```diff
@@ -0,0 +1,30 @@
+### RPM external pcm_util 1.0
+
+Source: none
+BuildRequires: root clhep tinyxml2
+
+%prep
+
+
+%build
+
+rm -f empty.h
+rm -f dummy.modulemap
+
+touch empty.h
+echo "module Dummy{}" > dummy.modulemap
+
+rm -f dummy_dict.cc
+rm -f libDummy.so
+rootcling dummy_dict.cc -moduleMapFile=${CLHEP_ROOT}/include/module.modulemap -s ./libDummy.so -moduleMapFile=dummy.modulemap -cxxmodule -m clhep -mByproduct clhep  -I ${CLHEP_ROOT}/include/ empty.h
+
+rm -f dummy_dict.cc
+rm -f libDummy.so
+rootcling dummy_dict.cc -s ./libDummy.so -moduleMapFile=dummy.modulemap -cxxmodule -m tinyxml2 -mByproduct tinyxml2  -I ${TINYXML2_ROOT}/include/ empty.h
+
+
+%install
+
+mkdir %{i}/include
+cp clhep.pcm %{i}/include/.
+cp tinyxml2.pcm %{i}/include/.
```

Comment:
Mostly PCM resides in the lib directory, so I would suggest to copy them to lib directory and use LIBDIR in toolfile too.

@vgvassilev , If I remember correctly, there was some restrictions about having libNAME.so and NAME.pcm in same directory (may be that is only for root dictionaries). So in case of these (clhep and tinyxml) PCMs do we need a library with same name in lib directory?

Reply:
No, they should not be connected to any library/dictionary.

Reply:
also I think llvm finds PCMs via CLING_PREBUILT_MODULE_PATH ... right @vgvassilev ? I guess we need to set CLING_PREBUILT_MODULE_PATH in toolfile too

Reply:
Good point, usually we do. In those particular case tinyxml2 and clhep depend only on libc and std (afaik) which will be loaded by default by rootcling -- we should be safe without that variable I think.

Reply:
without the variable how will llvm know about these pcms?

Reply:
for cmssw, we set CLING_PREBUILT_MODULE_PATH (pointing to `CMSSW_BASE/lib/arch:CMSSW_RELEASE/BASE/lib/arch` ) so that already build pcms are visible to clang.

Reply:
@smuzaffar - how do I add a PATH like variable to the tool file? Or does scram magically know that this variable is a PATH like variable?

Reply:
> without the variable how will llvm know about these pcms?

If you mean how rootcling will find libc.pcm and std.pcm -- you are right in that case we need that variable pointing to the location of those.

Reply:
> @smuzaffar - how do I add a PATH like variable to the tool file? Or does scram magically know that this variable is a PATH like variable?

just like https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_2_X/master/clhep-toolfile.spec#L20 i.e. add `type="path"`

I think you do not need LIBDIR or LD_LIBRARY_PATH for this tool. Just add
```
<runtime name="CLING_PREBUILT_MODULE_PATH" value="$PCM_UTIL_BASE/lib" type="path"/>
```